src/utils/notifications.utils.ts

```diff
@@ -5,8 +5,26 @@ interface INotification {
     type: string
 }
 
-export const sendNotification = async (notificationPayload: { accounts: string[]; notification: INotification }) => {
+export const sendNotification = async (
+    senderAddress: string,
+    recipientAddress: string | undefined,
+    chainName: string
+) => {
+    console.log('sendNotification', senderAddress)
+    const accounts = [`eip155:1:${senderAddress}` ?? '']
+    const notification: INotification = {
+        title: 'Peanut Protocol',
+        body: `Your link has been claimed on ${chainName} by ${recipientAddress ?? ''}`,
```

Comment:
This is not part of this PR, but imo I'd also add the USD amount / tokenamount to the message.

"your 10 usdc link ...."

Imagine if you sent multiple links, how does this tell you which one it is?

Reply:
Also, good on splitting this out, since we have multiple success views. Sweet.

Reply:
Yeah, agreed! I was thinking about this. Created a backlog task with owner = Bo